apollo-federation/src/subgraph/spec.rs

```diff
@@ -51,6 +51,7 @@ pub const REQUIRES_DIRECTIVE_NAME: Name = name!("requires");
 pub const SHAREABLE_DIRECTIVE_NAME: Name = name!("shareable");
 pub const TAG_DIRECTIVE_NAME: Name = name!("tag");
 pub const FIELDSET_SCALAR_NAME: Name = name!("FieldSet");
+pub const CONTEXTFIELDVALUE_SCALAR_NAME: Name = name!("ContextFieldValue");
```

Comment:
a typo?
```suggestion
pub const CONTEXT_FIELD_VALUE_SCALAR_NAME: Name = name!("ContextFieldValue");
```

Reply:
This case is [similar to above](https://github.com/apollographql/router/pull/6380#discussion_r1868170997), where we're following an existing convention (e.g. `FIELDSET_SCALAR_NAME` for `FieldSet`). Definitely down to change the convention across the crate in a follow-up.